### ISSUE TITLE
With the 3.0.0-beta.x code, page scrolling FPS slows down after hiding tooltip (60 goes down to 30 at best, 15 at worst)

### ISSUE THREAD

**Issue by BillyRayPreachersSon** [sir-dunxalot/ember-tooltips#236]
We'e got a page where the tooltip is shown based on a dynamic property, and hidden 1.5 seconds later using an `Ember.later` call.

To get a baseline for scrolling FPS without the tooltip, I refresh the page, start Chrome's profiling tools, and scroll up and down continuously (using the mouse wheel) for 5 seconds before stopping the profiler. This way, I get a consistent 57-60 FPS the whole time.

To test with the tooltip, I refresh the page again, show the tooltip, and wait for it to hide before starting the profiler and repeating the test.

This time, I can see a visible jank almost immediately. Stopping the profiler after 5 seconds shows me roughly 60 FPS for the first 500ms or so, and then it drops to anywhere between 15 and 30 FPS (although it looks really, really jittery). If I leave the profiler recording for 10 or 15 seconds, sometimes it'll crash and take the whole tab down with it.

According to the profiler, the culprit seems to be the vast amount of times that Popper is running its `requestAnimationFrame` call: 
```
this.scheduleUpdate = function () {
    return requestAnimationFrame(_this.update);
};
```

This call is not run at all before the tooltip is shown, so does it really need to be shown after the tooltip is shown and then hidden?

Granted, it might need to run while the tooltip is shown, to reposition it... but for the massive slowdown it causes, can the positioning calculations be disabled when the tooltip is hidden, and run again when shown?

Thanks!

**Comment by lan0** [comment]
Does https://github.com/sir-dunxalot/ember-tooltips/pull/235 fix your problem?

**Comment by BillyRayPreachersSon** [comment]
Sorry for the delay in replying - we ended up removing `ember-tooltips` and just rolling our own, as we only needed very basic functionality.

**Comment by ivanvanderbyl** [comment]
I can confirm that #235 fixes the problem, please merge.

**Comment by maxfierke** [comment]
The slowdown/100% CPU issue should be fixed in 3.0.0-beta.3, though I've observed that sometimes the framerate will still dip in some cases. I'll keep looking into that.

**Comment by maxfierke** [comment]
More robustly addressed in `3.0.0-beta.4`, but potentially some mor e optimizations possible (should maintain ~60fps when scrolling though)

**Comment by maxfierke** [comment]
I'm going to close this, because I think this is pretty well addressed in 3.0.0-beta.4, but feel free to ping me to re-open if this is still a problem.
